tests/test_revision.py

```diff
@@ -125,6 +129,39 @@ def test_get_revision_heads_multiple(self):
             "heads",
         )
 
+    def test_get_revisions_head_multiple(self):
+        map_ = RevisionMap(
+            lambda: [
+                Revision("a", ()),
+                Revision("b", ("a",)),
```

Comment:
**Koichiro Den** (lkpdn) wrote:

Ack

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/alembic/+/2379